hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1601,6 +1601,16 @@
    */
   public static final int BATCH_ROWS_THRESHOLD_DEFAULT = 5000;
 
+  /**
+   * List of column families that cannot be deleted from the hbase:meta table.
+   * They are critical to cluster operation. This is a bit of an odd place to
+   * keep this list but then this is the tooling that does add/remove. Keeping
+   * it local!
```

Comment:
This comment is wrong, right? We don't have any tooling in HConstants. 

This should go where table descriptors are defined and managed.

Reply:
Ok, will move it back to ModifyTableProcedure